test/test-virtual-rules.js

```diff
@@ -0,0 +1,23 @@
+var path = require('path');
+var assert = require('chai').assert;
+var glob = require('glob');
+var axe = require(path.join(__dirname, '../axe'));
```

Comment:
Isn't this the same thing?

```suggestion
var axe = require('..');
```

Reply:
`axe` isn't a folder so it's the `axe.js` file we're after

Reply:
[`axe.js` is set to `main` in `package.json`](https://github.com/dequelabs/axe-core/blob/develop/package.json#L51).  You don't need anything other than `..`. I'm 👍 for removing the `__dirname` though.

Reply:
Ah. I didn't know that. I knew it looked for an `index` file when using a directory, but I didn't know a relative file path would look at `pacakge.json` as well (I only thought that was for module names).